apps/pie-docs/src/components/notification/overview/overview.md

```diff
@@ -0,0 +1,401 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Notification
+    order: 1
+eleventyComputed:
+    usage: "{% include './usage.json' %}"
+shouldShowContents: true,
+---
+
+## Overview
+
+The purpose of notifications is to promptly inform users about relevant information or changes that require their attention. Notifications can be used to communicate messages like new messages, system updates, completed actions, errors or reminders.Notifications often include options for users to interact with the displayed information, such as dismissing the notification, taking action or accessing related content. They play a crucial role in keeping users informed and engaged with the application or system.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/overview.svg",
+    alt: "An informational notification with a single button"
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use the component to provide feedback on the user’s status, outcome of an action, or info on a task.  "
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Use notification pinned to the top of the interface. See below for full usage guidelines between notification and toast."
+        ]
+    }
+} %}
+
+---
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/anatomy.svg",
+    alt: "Anatomy of a notification that contains a leading icon, title and supporting text as well as dual actions and a dismiss button.",
+    width: 1024,
+    variant: "secondary"
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Icon (Optional):** Leading icon that visually reinforces the content.",
+        "**Title (Optional)**: Provides overall content of the content.",
+        "**Supporting text**: Provides supporting information to the user.",
+        "**Close (Optional)**: Let the user dismiss the notification.",
+        "**Actions (Optional)**: Button options let the user proceed with or dismiss the notification."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Neutral
+
+Neutral  provides generic messages to users that may not be related to the current action or task they’re performing. Does not require immediate action and can be dismissed on a timer or persist, depending on the content.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/variation-neutral.svg",
```

Comment:
Is the image correct? On the designs it has a white background over the pie-notification, not grey